### ISSUE TITLE
cfn-cli no longer working with latest version of awscli (version 1.16.26)

### ISSUE THREAD

**Issue by ouelletandrebni** [Kotaimen/awscfncli#50]
Hi,

**Workaround**

If you are using virtualenv, you can specify the following versions in requirements.txt.  This will fix this issue for now.

```
awscfncli2==2.0.1
awscli==1.16.21
boto3==1.9.11
botocore==1.12.11
```

If you are using user level or system level python packages, running the following commands will fix this issue for now. 

```
pip install awscfncli2==2.0.1
pip install awscli==1.16.21
pip install boto3==1.9.11
pip install botocore==1.12.11
```

**Issue**

Using awscfncli 2.0.1 has been working great, until recently when my version of awscli got upgraded to the latest version of 1.16.**26**  Now awscfncli is throwing the following error (see below)

It was working ok at least until awscli 1.16.**21**

```
#$ cfn-cli --profile myprofile -f cfn/mycfnfile.yml status

Traceback (most recent call last):
  File "/home/user/workspace/myproject/.venv/bin/cfn-cli", line 7, in <module>
    from awscfncli2.__main__ import main
  File "/home/user/workspace/myproject/.venv/lib/python3.6/site-packages/awscfncli2/__main__.py", line 26, in <module>
    from .cli import cfn_cli
  File "/home/user/workspace/myproject/.venv/lib/python3.6/site-packages/awscfncli2/cli/__init__.py", line 2, in <module>
    from .main import cfn_cli
  File "/home/user/workspace/myproject/.venv/lib/python3.6/site-packages/awscfncli2/cli/main.py", line 10, in <module>
    from .utils.context import ContextObject
  File "/home/user/workspace/myproject/.venv/lib/python3.6/site-packages/awscfncli2/cli/utils/__init__.py", line 8, in <module>
    from .package import run_packaging
  File "/home/user/workspace/myproject/.venv/lib/python3.6/site-packages/awscfncli2/cli/utils/package.py", line 10, in <module>
    from awscli.customizations.cloudformation.artifact_exporter import Template, \
ImportError: cannot import name 'EXPORT_DICT'
```

**Comment by GlieseRay** [comment]
@ouelletandrebni Thanks for your feedback. This may be caused by the upgrading of awscli, we will schedule a hotfix for that.

**Comment by Kotaimen** [comment]
Also need set up an auto build to watching new cfncli versions.

**Comment by Kotaimen** [comment]
@GlieseRay Please release a hot fix in 2.0.x

**Comment by Kotaimen** [comment]
Hi @ouelletandrebni,
I have pinpointed the change which breaks `cfncli`: https://github.com/aws/aws-cli/commit/344560043407f18098bfaec3e8bab50b3b115f76, note the change is place after `1.16.36`, not `1.16.26`.

I'll make the fix to develop branch and @GlieseRay please make a hot fix on `2.0.x`.

**Comment by Kotaimen** [comment]
Hi @ouelletandrebni, this issue is fixed in `2.0.3`:
```bash
$ cfn-cli --version
cfn-cli, version 2.0.3
$ aws --version
aws-cli/1.16.39 Python/3.7.0 Darwin/18.0.0 botocore/1.12.29
```

**Comment by Kotaimen** [comment]
@GlieseRay It seems this is not easily fixed:
```
Traceback (most recent call last):
  File "/usr/local/bin/cfn-cli", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/awscfncli2/__main__.py", line 31, in main
    auto_envvar_prefix='CFN'
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/awscfncli2/cli/utils/deco.py", line 16, in wrapper
    return f(ctx, *args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/awscfncli2/cli/stack/update.py", line 41, in update
    ignore_no_update, override_policy)
  File "/usr/local/lib/python3.7/site-packages/awscfncli2/cli/stack/update.py", line 52, in update_one
    run_packaging(stack_config, session, ctx.obj.verbosity)
  File "/usr/local/lib/python3.7/site-packages/awscfncli2/cli/utils/package.py", line 43, in run_packaging
    prefix=stack_config['StackName'])
  File "/usr/local/lib/python3.7/site-packages/awscfncli2/cli/utils/package.py", line 97, in package_template
    exported_template = template.export()
  File "/usr/local/lib/python3.7/site-packages/awscli/customizations/cloudformation/artifact_exporter.py", line 513, in export
    if exporter_class.RESOURCE_TYPE != resource_type:
AttributeError: 'str' object has no attribute 'RESOURCE_TYPE'
```

**Comment by GlieseRay** [comment]
I am fixing, something wrong happens....

**Comment by GlieseRay** [comment]
Fixed in 2.0.4

**Comment by Kotaimen** [comment]
Hi @GlieseRay:

It seems `awscli` changed their packing and broke us, again:

```
 cfn-cli -s 'Dev.Sandbox01' stack update
Traceback (most recent call last):
  File "/Users/lcao/Documents/dev/workspace/awscfncli/awscfncli2/runner/runbook/package.py", line 18, in <module>
    from awscli.customizations.cloudformation.artifact_exporter import \
ImportError: cannot import name 'EXPORT_LIST' from 'awscli.customizations.cloudformation.artifact_exporter' (/usr/local/lib/python3.7/site-packages/awscli/customizations/cloudformation/artifact_exporter.py)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/cfn-cli", line 11, in <module>
    load_entry_point('awscfncli2', 'console_scripts', 'cfn-cli')()
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 487, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2728, in load_entry_point
    return ep.load()
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2346, in load
    return self.resolve()
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2352, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/Users/lcao/Documents/dev/workspace/awscfncli/awscfncli2/__main__.py", line 26, in <module>
    from .cli import cfn_cli
  File "/Users/lcao/Documents/dev/workspace/awscfncli/awscfncli2/cli/__init__.py", line 2, in <module>
    from .main import cfn_cli
  File "/Users/lcao/Documents/dev/workspace/awscfncli/awscfncli2/cli/main.py", line 10, in <module>
    from .context import ClickContext
  File "/Users/lcao/Documents/dev/workspace/awscfncli/awscfncli2/cli/context.py", line 3, in <module>
    from ..runner import Boto3Profile, Boto3RunBook, StackSelector
  File "/Users/lcao/Documents/dev/workspace/awscfncli/awscfncli2/runner/__init__.py", line 2, in <module>
    from .runbook import Boto3Profile, Boto3RunBook
  File "/Users/lcao/Documents/dev/workspace/awscfncli/awscfncli2/runner/runbook/__init__.py", line 3, in <module>
    from .boto3_runbook import Boto3RunBook
  File "/Users/lcao/Documents/dev/workspace/awscfncli/awscfncli2/runner/runbook/boto3_runbook.py", line 5, in <module>
    from .boto3_context import Boto3DeploymentContext
  File "/Users/lcao/Documents/dev/workspace/awscfncli/awscfncli2/runner/runbook/boto3_context.py", line 10, in <module>
    from .package import package_template
  File "/Users/lcao/Documents/dev/workspace/awscfncli/awscfncli2/runner/runbook/package.py", line 22, in <module>
    from awscli.customizations.cloudformation.artifact_exporter import \
ImportError: cannot import name 'EXPORT_DICT' from 'awscli.customizations.cloudformation.artifact_exporter' (/usr/local/lib/python3.7/site-packages/awscli/customizations/cloudformation/artifact_exporter.py)
```

Please do a hot fix in `2.1` and `2.0`, thanks.

**Comment by Kotaimen** [comment]
Fixed in `2.1.1`
